extensions/git/src/repository.ts

```diff
@@ -1869,7 +1901,10 @@ export class Repository implements Disposable {
 		}
 	}
 
-	private async run<T>(operation: Operation, runOperation: () => Promise<T> = () => Promise.resolve<any>(null)): Promise<T> {
+	private async run<T>(
+		operation: Operation,
+		runOperation: () => Promise<T> = () => Promise.resolve<any>(null),
+		optimisticResourceGroups: () => GitResourceGroups | undefined = () => undefined): Promise<T> {
```

Comment:
Should this be called `getOptimisticResourceGroups` instead?

Reply:
Fixed.